Orange/widgets/visualize/owdistributions.py

```diff
@@ -16,13 +15,7 @@
 from Orange.widgets.utils import itemmodels, colorpalette
 from Orange.widgets.widget import InputSignal
 
-
-def is_discrete(var):
-    return isinstance(var, Orange.data.DiscreteVariable)
-
-
-def is_continuous(var):
-    return isinstance(var, Orange.data.ContinuousVariable)
+import pyqtgraph as pg
```

Comment:
Why move pyqtgraph imports bellow Orange ones? Pyqtgraph is external library, same as numpy (and PyQt).